source/cmake/CompilerOptions.cmake

```diff
@@ -65,6 +66,7 @@ elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
             4242 # Narrowing conversions: Too strict and noisy for this code base
             4820 # The type and order of elements caused the compiler to add padding to the end of a struct
             4868 # compiler may not enforce left-to-right evaluation order in braced initializer list
+            4355 # used in base member initializer list
```

Comment:
Can you include the full name of this warning here? I think this is about `this` being used in the base member initializer list, and currently it doesn't show.

Reply:
https://github.com/zivid/zivid-cpp-samples/pull/169/commits/47f2c0e7541410eabae854a9592eefb1538adfc7